internal/index/indexscanner/testdata/testindex/receipts/default-explicit.yaml

```diff
@@ -0,0 +1,35 @@
+# Copyright 2019 The Kubernetes Authors.
```

Comment:
I must admit, I kinda dislike we add 3 nearly identical test files just for this. :(
But I see no other way to test this either.